### PR TITLE
1.3 grammar style on communication

### PR DESCRIPTION
**Pull request by andyjohnson**
I realized I didn't have a new branch when I started editing the first lines and navigated away, figuring it would cancel.  That first commit may be that.  I simply repeated in the next commit, so hopefully nothing wonky happened in there.  Look in each commit for notes.  Thanks!

### CODE REVIEW

File: xAPI-Communication.md [hunk 1]
```diff
@@ -439,8 +446,8 @@ syntax after this represents the particular Resource used. A full list of the re
 described in [Section 6.4.2 OAuth Authorization Scope](#oauthscope).
 * <a name="2.0.s1.b3"></a>The LRS MAY support additional resources not described in this specification. 
 * <a name="2.0.s1.b4"></a>Past, current and future versions of this specification do not and will not define resource endpoints 
-with path segments starting 'extensions/'. LRSs supporting additional resources not defined 
-in this specification SHOULD define their endpoints with path segments starting with 'extensions/'.
+with path segments starting with `extensions/`. LRSs supporting additional resources not defined in this specification SHOULD 
+define their endpoints/resource locations with path segments starting with `extensions/`.
```
Comment (garemoko) [inline xAPI-Communication.md]: We should either call it endpoints or resource locations, not both. I prefer the original 'endpoints'.

File: xAPI-Communication.md [hunk 2]
```diff
@@ -506,7 +513,7 @@ Stores a Statement, or a set of Statements.
 
 **Content:** An array of Statements or a single Statement to be stored. 
 
-**Returns:** `200 OK`, Array of Statement id(s) (UUID) in the same order as the corresponding stored statements.  
+**Returns:** `200 OK`, Array of Statement ids (UUID) in the same order as the corresponding stored Statements.
```
Comment (garemoko) [inline xAPI-Communication.md]: id(s) is more technically correct here. There may be only one.

File: xAPI-Communication.md [hunk 3]
```diff
@@ -702,34 +708,33 @@ retrieve them will be included in the StatementResult Object.
...
 
 * <a name="2.1.3.s2.b4"></a>In the event that no Statements are found matching the query filter criteria, the LRS MUST still return 
-`HTTP 200` and a [StatementResult](#retstmts) Object. In this case, the "statements" property will contain
-an empty array.
+`200 OK` and a [StatementResult](#retstmts) Object. In this case, the "statements" property will contain an empty array.
 
 * <a name="2.1.3.s2.b5"></a>The LRS MUST include the header "X-Experience-API-Consistent-Through", in 
 [ISO 8601 combined date and time](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations) format, 
-on all responses to Statements requests, with a value of the Timestamp for which all Statements that have or will have a 
+on all responses to Statements Resource requests, with a value of the timestamp for which all Statements that have or will have a
```
Comment (garemoko) [inline xAPI-Communication.md]: good catch on timestamp

File: xAPI-Communication.md [hunk 4]
```diff
@@ -1276,45 +1283,45 @@ Stores, changes, fetches, or deletes the specified profile document in the conte
...
 Example resource endpoint: http://example.com/xAPI/activities/profile
 
-Fetches Profile Ids of all profile entries for an Activity. If "since" parameter is specified, this is limited to 
+Fetches profile ids of all profile entries for an Activity. If "since" parameter is specified, this is limited to 
 entries that have been stored or updated since the specified Timestamp (exclusive).  
 
 **Content:** None.
 
-**Returns:** `200 OK`, Array of Profile Ids  
+**Returns:** `200 OK`, Array of profile ids
```
Comment (garemoko) [inline xAPI-Communication.md]: See comment above about statement id(s). There could also be 1 profile id returned here.

**Comment by garemoko** [comment]
Added a couple of comments, otherwise looks good.

**Comment by andyjohnson** [comment]
Ok, fixed the comments and a couple stray section references (making a new issue on this).  Most of #973 is now in here - just have to apply to the rest of the parts.

File: xAPI-Communication.md [hunk 5]
```diff
@@ -2080,12 +2100,12 @@ Converted to 1.0.0:
 
 <a name="Appendix3B"/>
 
-### <a name="B">Appendix B</a>: Table of All Endpoints
+### <a name="B">Appendix B</a>: Table of All Resources
 
-### <a name="B.s1"></a>Required Resource Endpoints
+### <a name="B.s1"></a>Required Endpoint
```
Comment (garemoko) [inline xAPI-Communication.md]: I don't think this heading makes sense singular. What are you trying to achieve here?

File: xAPI-Communication.md [hunk 6]
```diff
@@ -1106,7 +1114,7 @@ about the Agent it received in the request.
 	<tr id="2.4.s5.table1.row1">
 		<td>objectType</td>
 		<td>String</td>
-		<td>"Person"</td>
+		<td>`Person`</td>
```
Comment (garemoko) [inline xAPI-Communication.md]: You missed a bit. :-)

File: xAPI-Communication.md (hunk 5 above)
Comment (andyjohnson) [inline xAPI-Communication.md]: Typo :)  Fixing

Comment (andyjohnson) [inline xAPI-Communication.md]: And actually, looking at it - it is Resources as the table has the Endpoint and Function

**Comment by garemoko** [comment]
Checked the specific things I flagged last time and gave the rendered version a quick glance over. 

+1

File: xAPI-Communication.md [hunk 7]
```diff
@@ -248,58 +250,62 @@ Strongly consider security risks before making the decision to use implementatio
...
+* <a name="1.5.1.s1.b1"></a>When receiving a PUT or POST with a document type of `application/json`, an LRS MUST accept batches 
+of Statements which contain no Attachment Objects.
+* <a name="1.5.1.s1.b2"></a>When receiving a PUT or POST with a document type of `application/json`, an LRS MUST accept batches 
+of Statements which contain only Attachment Objects with a populated fileUrl.
 
 <a name="multipartmixed"/> 
 #### <a name="1.5.2">1.5.2</a> Multipart/Mixed
 
-The multipart/mixed content type is used for requests that *could* include Attachments. This does not mean that all 
-multipart/mixed requests neccesarily do include Attachments.
```
Comment (ljwolford) [inline xAPI-Communication.md]: spelling, necessarily

File: xAPI-Communication.md [hunk 8]
```diff
@@ -248,58 +250,62 @@ Strongly consider security risks before making the decision to use implementatio
...
+The `multipart/mixed` content type is used for requests that *could* include Attachments. This does not mean that all 
+"multipart/mixed" requests neccesarily do include Attachments.
```
Comment (ljwolford) [inline xAPI-Communication.md]: spelling, necessarily

Comment (andyjohnson) [inline xAPI-Communication.md]: Good catch!

**Comment by ljwolford** [comment]
Gave it a quick read through, didn't notice anything glaring. +1

**Pull request merged**